app/assets/javascripts/specific/main_menu.js.erb

```diff
@@ -63,23 +46,18 @@ jQuery(document).ready(function($) {
     return this;
   };
 
-  var toggler = $('<a class="toggler" href="#"><i class="icon6 icon-toggler icon-arrow-down1" aria-hidden="true"></i><span class="hidden-for-sighted"></span></a>')
+  var toggler = $('<a class="toggler" href="#"><i class="icon6 icon-toggler icon-arrow-right3" aria-hidden="true"></i><span class="hidden-for-sighted"></span></a>')
     .click(function(event) {
       var target = $(this);
-
       if (target.hasClass('toggler')) {
-        var menuItemWrapper = target.parent();
-        var menuParent = menuItemWrapper.toggleClass('open').parent().find('ul').not('ul ul ul');
-
-        menuParent.mySlide();
-        if (menuItemWrapper.hasClass('open')) {
-          menuParent.show();
-          menuParent.find('li > a:first').focus();
-        } else {
-          menuParent.hide();
-        }
-
-        setExpandStatus(target);
+
+        // TODO: Instead of hiding the sidebar move sidebar's contents to submenus and cache it.
```

Comment:
Are these TODOs out of scope for this PR?

Reply:
Yes, out of scope, but wanted.